### ISSUE TITLE
Cross-refference Greenfield API view in the Legacy API view

### ISSUE THREAD

**Issue by pavlenex** [btcpayserver/btcpayserver#5571]
**Context:**

When started, BTCPay Server had a BitPay-compatible API, which to this day still works. However, due to various limitation of that API, and BTCPay growing as a stack, we introduced [Greenfield API ](https://docs.btcpayserver.org/API/Greenfield/v1/) which allows pretty much to run BTCPay headlessly. 

The difference between the two API's is that Greenfield API is account based, where as, BitPay compatible (legacy) API is store based. This is currently reflected in BTCPay's UI, where Store Settings > Access Tokens showcases Legacy (BitPay compatible API), and Account > Manage > API Keys is for Greenfield. 

**Problem**:

While we mention (link to) the legacy api on the Account > Manage > API Keys, we do not do the same on the Legacy API page in Store Settings > Access Tokens, which often leads people using Greenfield to enter legacy API keys and face problems.
![Screenshot 2023-12-20 at 11 51 51](https://github.com/btcpayserver/btcpayserver/assets/36959754/6f5211fc-a49e-4773-b8fe-d50c0435ce33)

**Proposed solution:**

I think besides simple linking to the Account Manage API Keys page, we may also want to add a bit of context so developers building on top of BTCPay don't face this barriers from the get go.

```
"On this page  you can manage Legacy (BitPay compatible) API keys, which are store-based. For the new Greenfield API, please generate your API keys [here]."
```

https://github.com/btcpayserver/btcpayserver/assets/36959754/e07cbd5a-39db-4e58-885b-34185b38cdb7

**Before doing a PR,** it would be good if we can first discuss the proposed copy and see if it needs to be ironed out. And how specifically we want to add this text according to our style guidelines cc @dstrukt

**Comment by hiluan** [comment]
> While we mention (link to) the legacy api on the Account > Manage > API Keys, we do not do the same on the Legacy API page in Store Settings > Access Tokens, which often leads people using Greenfield to enter legacy API keys and face problems.

I'm experiencing the same issue. 

I've been struggling to use the legacy API instead of the Greenfield API. I kept wondering why it wasn't working. The Greenfield API documentation doesn't mention the need to go to Account > Manage > API keys. I initially thought the legacy API was the same as the Greenfield API until I came across this.

> Before doing a PR, it would be good if we can first discuss the proposed copy and see if it needs to be ironed out. And how specifically we want to add this text according to our style guidelines cc @dstrukt

I want to help with this issue. Who should I talk to about it??

**Comment by pavlenex** [comment]
We can discuss possible improvements here @hiluan. 

> Greenfield API documentation doesn't mention the need to go to Account > Manage > API keys

This is problematic and should be addressed imo. Also related https://github.com/btcpayserver/btcpayserver/issues/5572

Would be at least useful to note all of this struggles you're facing, and then let's figure out how to improve that experience.

**Comment by hiluan** [comment]
It would be great to have a direct link or button under Store Settings > Access Tokens for creating a Greenfield API Key, like "For Greenfield API Key, click here." Alternatively, a button like "Generate Greenfield API keys" directly in that section could work.

If a dedicated button isn't possible, please guide users on how to generate Greenfield API keys by adding a clear reference in the Greenfield API documentation, such as 'Generate a Greenfield API Key: go to Account > Manage > API keys.'

I finally got the right API key today from "Account > Manage > API keys," and it's working well. However, to improve user experience, in the Greenfield doc, it would be helpful to include sample codes for different languages, not just responses. For example:
![Screenshot 2023-12-26 193451](https://github.com/btcpayserver/btcpayserver/assets/76276426/c5f48432-cdff-4125-beb5-da88b446812b)
https://finnhub.io/docs/api/company-basic-financials

![Screenshot 2023-12-26 193236](https://github.com/btcpayserver/btcpayserver/assets/76276426/938bdd18-0094-4ad8-a313-5eebfd5494e4)
https://stripe.com/docs/upgrades/manage-payment-methods

**Comment by hiluan** [comment]
I made a small change by adding this:
![Screenshot 2024-01-01 203900](https://github.com/btcpayserver/btcpayserver/assets/76276426/ce9f8821-d689-4a4c-a2b3-2a7af145558f)

Let me know what you think. Thanks!

**Comment by Kukks** [comment]
fixed in #5635
